test_util/launch_configs/aws-zen-cf.yaml

```diff
@@ -0,0 +1,19 @@
+---
+deployment_type: template
+deployment_name: This-is-a-test-stack
+template_url: https://s3.amazonaws.com/downloads.dcos.io/dcos/testing/master/cloudformation/coreos-zen-1.json
+provider: aws
+provider_info:
+    region: us-west-2
+    access_key_id: ALSAFLAJ4L20922KF20TL294
+    secret_access_key: ADWOFIAOFWJTPW3924JR
+key_helper: true
+zen_helper: true
```

Comment:
I was putting it at top-level because its a dcos-launch option, and not a native provider option, but at the end of the day, this is a provider API wrapper, so your argument seems good to me